ci-operator/step-registry/redhat-chaos/kubevirt-outage/redhat-chaos-kubevirt-outage-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o errexit
+
+console_url=$(oc get routes -n openshift-console console -o jsonpath='{.spec.host}')
+export HEALTH_CHECK_URL=https://$console_url
+oc get vmis -A
+set -o nounset
+set -o pipefail
+set -x
+
+
+ES_PASSWORD=$(cat "/secret/es/password")
+ES_USERNAME=$(cat "/secret/es/username")
+
+export ES_PASSWORD
+export ES_USERNAME
+
+export ES_SERVER="https://search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
+
+echo "kubeconfig loc $$KUBECONFIG"
+echo "Using the flattened version of kubeconfig"
+oc config view --flatten > /tmp/config
+
+export KUBECONFIG=/tmp/config
+
+export KRKN_KUBE_CONFIG=$KUBECONFIG
+export ENABLE_ALERTS=False
+telemetry_password=$(cat "/secret/telemetry/telemetry_password")
+export TELEMETRY_PASSWORD=$telemetry_password
+
+export NAMESPACE=$TARGET_NAMESPACE 
+
+oc get vmis -A 
+
+export KUBE_VIRT_NAMESPACE=$TARGET_NAMESPACE
+./kubevirt-outage/prow_run.sh
+rc=$?
```

Comment:
Since the script is running with `set -e`, non-0 exit code from execution of `./kubevirt-outage/prow_run.sh` will stop the script. So really printing on line 42 will never occurs in case of that script execution exit with error. If it's printed, it will always be `0` (successful).

If the execution of line 38-40 is desired regardless of the exit status of `./kubevirt-outage/prow_run.sh`, the do the following instead:
```suggestion
./kubevirt-outage/prow_run.sh || rc=$?
```
The above will not stop the script and assign the var. `rc` to the exit status of `./kubevirt-outage/prow_run.sh` execution (and this particular assignment will always be successful). Then line 38-40 will be executed, even the `./kubevirt-outage/prow_run.sh` return an error and it make sense to print the `rc` on line 42.